src/sentry/integrations/jira/client.py

```diff
@@ -16,59 +16,6 @@
 CUSTOMFIELD_PREFIX = "customfield_"
 
 
-class JiraCloud:
```

Comment:
This class was removed in the refactor to get rid of `jira_style` which was used to determine whether it was Jira Cloud or Jira Server.